lib/common/output_log.c

```diff
@@ -329,6 +329,18 @@ pcmk__mk_log_output(char **argv) {
     return retval;
 }
 
+int
+pcmk__output_get_log_level(pcmk__output_t *out)
```

Comment:
could be const

Reply:
Yep thanks. I'm gonna pull this commit out of this PR, in case I don't end up using it. I'm 90% sure I will, but the formatted output of XML and patchsets is messy.